atomic_reactor/plugins/pre_fetch_sources.py

```diff
@@ -408,8 +408,12 @@ def _create_full_remote_sources_map(self, request_session, remote_sources_map,
 
     def _check_if_package_excluded(self, packages, denylist_sources, remote_archive):
         # check if any package in cachito json matches excluded entry
+        # strip leading os.sep as package names can include git path with '/' before package name
+        # or just package name, or package name with leading '@' depending on package type
+        denylist_packages = [k[1:] for k in denylist_sources]
```

Comment:
How about changing this in the __get_denylist_sources_  method where we adding the os.separator just to remove it later?

Reply:
it is used in 2 places, for checking package name and for excluding path (this uses leading separator)

Reply:
Should we use `lstrip` to better convey the meaning?
```python
denylist_packages = [k.lstrip('/') for k in denylist_sources]
```

Reply:
sure I can do that